integrations/anthropic/integration.definition.ts

```diff
@@ -3,11 +3,14 @@ import { modelId } from 'src/schemas'
 
 export default new IntegrationDefinition({
   name: 'anthropic',
-  version: '0.2.0',
+  version: '0.3.0',
   readme: 'hub.md',
   icon: 'icon.svg',
   entities: {
     model: {
+      schema: interfaces.llm.entities.model.schema,
+    },
+    modelRef: {
       schema: z.object({
         id: modelId,
```

Comment:
`modelRef` is for specifying just the model ID when calling `generateContent` (due to an agreed convention of using object entities for this kind of things), while `model` is the entity that represents the full detals of a given model.

Reply:
This is not what entities are for; you should think entities as type arguments. In this case, you use the same schema for every integration

Reply:
@allardy I spoke with Fleur and he clarified the use of entities to be for type arguments rather than all entities in general, so I refactored the LLM interface and integrations to only need to use the `modelRef` entity, while allowing each integration to return the list of models in a type safe way by intersecting the modelRef (type argument) with the `ModelSchema` in the LLM interface. Details in this commit: https://github.com/botpress/botpress/pull/13180/commits/c426ed7f17bf8e6935ec973325c2097f867519d3